### ISSUE TITLE
Determine our Maths/LaTeX strategy

### ISSUE THREAD

**Issue by rgbkrk** [nteract/archived-desktop-app#99]
We likely need to support the current notebook's `$` inline and `$$` block modes via KaTeX and/or MathJax.

Resources:
- [Discussion on a react notebook renderer](https://github.com/plotly/react-ipython-notebook/issues/6#issuecomment-171255664)
- [Khan Academy's TeX component](https://github.com/Khan/react-components/blob/master/js/tex.jsx)

**Comment by mangecoeur** [comment]
Would it be possible to share the regular notebook's approach? They use MathJax - they make it simpler by only including the parts they need. Turns out if you drop legacy browser support you don't need much of the project - I did something similar with my Atom package  [preview-inline](https://atom.io/packages/preview-inline).

I definitely do NOT recommend KaTeX - it's fine when you know what maths you will render and plan to keep it simple, but when you want to allow people to write arbitrary math you quickly run into its limitations. I used to include it in preview-inline with a fallback to MathJax, but I found that when you are running locally it doesn't actually make much difference on speed and you end up with the extra complexity of two math rendering libraries
